### ISSUE TITLE
C7 data quality from summer 2018

### ISSUE THREAD

**Issue by kellyloria** [kellyloria/GL4_buoy_QAQC#2]
Seems to be a stage change between early and late season data for summer 2018 deployment. So need more data to better correlate how to transform raw C7 output to fall with in a acceptable range of chlor-a extractions. 

Could be that transformed 0 values in early season are wrong. 

The miniPAR sensor attached directly next to the C7 shows signs of movement and in the Y and Z acceleration graphs in the early part of the season (2018-07-03 to  07-10) where those values seem to be different do to sensor angle so it is likely that the buoy may have been moved. 

Looking over field observations the surface float became entangled with the lake tag line during between our 1st visit (2018-07-03 ) and our second visit (2018-07-10) could've forced the entire buoy to list to an angle, causing variation in sensor position and optical sensor values during this time.

**Comment by kellyloria** [comment]
Evidence of PAR:
[Summer2018_PAR_check_X.pdf](https://github.com/kellyloria/GL4_buoy_QAQC/files/2865912/Summer2018_PAR_check_X.pdf)
[Summer2018_PAR_check_Y.pdf](https://github.com/kellyloria/GL4_buoy_QAQC/files/2865915/Summer2018_PAR_check_Y.pdf)
[Summer2018_PAR_check_Z.pdf]
(https://github.com/kellyloria/GL4_buoy_QAQC/files/2865916/Summer2018_PAR_check_Z.pdf)
[Summer2018_PAR_oddvalue.pdf](https://github.com/kellyloria/GL4_buoy_QAQC/files/2865919/Summer2018_PAR_oddvalue.pdf)

**Comment by kellyloria** [comment]
Created a flag "q" to show the early season chl-a data for the summer 2018 deployment looked wrong.

It is possible that the reflectance off of new bolts in the initial deployment (summer 2018) near the C7 (which was deployed without a radiation shield) caused the C7 sensor to read unreal values. Likely, once biofilm grew on the rest of the buoy it muted the reflectance. In summer of 2019 we created a radiation shield out of PCV and believe we corrected any issues.
